websites/A/AniWorld/presence.ts

```diff
@@ -51,71 +51,77 @@ presence.on(
 presence.on("UpdateData", async () => {
 	const page = document.location.pathname,
 		presenceData: PresenceData = {
-			largeImageKey: "anicloud",
+			largeImageKey: "aniworld-logo",
 			startTimestamp: Math.floor(Date.now() / 1000)
 		};
 	if (page === "/") presenceData.details = "Betrachtet die Startseite";
 	else if (page.startsWith("/anime/")) {
-		presenceData.details = `${
-			(document.querySelector("h1") as HTMLElement).textContent
-		}`;
-		presenceData.state = `${(
-			document.querySelector("title") as HTMLElement
-		).textContent
-			.split("Staffel")[0]
-			.replace("Filme von", " ")
-			.split(" | AniCloud.io - Animes gratis online ansehen")} - ${
-			(document.querySelector("h2") as HTMLElement).textContent
-		}`;
-		[presenceData.startTimestamp, presenceData.endTimestamp] =
-			presence.getTimestamps(currentTime, timeEnd);
-		presenceData.buttons = [
-			{
-				label: "Watch Anime",
-				url: document.location.href
-			}
-		];
+		// Check if we are on the episode selection page
+		if (page.split("/").length === 4) {
+			presenceData.details = `${
+				document.querySelector<HTMLHeadingElement>("h1").textContent
+			}`;
```

Comment:
This is already a string
```suggestion
			presenceData.details =
				document.querySelector<HTMLHeadingElement>("h1").textContent;
```